setup.py

```diff
@@ -4,7 +4,7 @@
 
 
 INSTALL_REQUIREMENTS = [
-    "Django>=1.11,<3.0",
+    "Django>=1.11,<4.0",
```

Comment:
@marktsavage Yup, well spotted. Not sure if you're aware of the suggestions feature of Github, it can help make getting your point or instruction across.

```suggestion
    "Django>=2.2,<4.0",
```